### ISSUE TITLE
Saving images doesn't work as of version 2.1.0

### ISSUE THREAD

**Issue by MirceaKitsune** [mcdamo/tab-image-saver#18]
As of version 2.1.0, which Firefox forcefully upgraded me to, the addon simply doesn't work any more. Whenever I try to save any images, the icon shows the addon working for a bit then says 0, despite having tabs with images beyond the minimum resolution setting. If I enable "Show notification popups" I get the error `Failed: 1` in the system tray.

Firefox 61.0.2 64bit, Linux openSUSE Tumbleweed x64 KDE, Tab Image Saver 2.1.0.

**Comment by mcdamo** [comment]
Look in the Browser Console for a verbose error message (I think Ctrl+Shift+J will open it).

**Comment by MirceaKitsune** [comment]
Yes, it opens. I'm seeing a reoccurring and unsurprisingly absurd message, which gets printed whenever I try to save the images:

`Sending message that cannot be cloned. Are you trying to send an XPCOM object? ExtensionUtils.jsm:546:50`

**Comment by MirceaKitsune** [comment]
Thank you kindly for the fix. The 2.1.1 update solves image saving, it now works as intended again and I'm seeing no more errors! Only remaining issue which seems to be related is that choosing the Downloads Folder option doesn't work, it fails and complains with the message:

`Unable to run the command specified. The file or folder does not exist.`

**Comment by nathan-whittington** [comment]
I'm having issues too.  With version 2.1.1 from the browser console log:

```
Invalid method for executeTabs: undefined background.js:474:9
executeTabs
moz-extension://06e97883-da6b-43f3-94e8-ec918bdcdad4/background/background.js:474:9
run
moz-extension://06e97883-da6b-43f3-94e8-ec918bdcdad4/background/background.js:651:15
```

**Comment by nathan-whittington** [comment]
I have it set to 'Right of Active tab' and 'include active tab is' unchecked.  Just now, when I tested it again, it worked.  I'm not doing anything different today, maybe restarting the browser fixed things?

**Comment by mcdamo** [comment]
Issue resolved in v2.1.1
